api/db/changelog/db.changelog-266-update-user-initial-credits-expiration-to-365-days.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="erollins" id="db.changelog-266-update-user-initial-credits-expiration-to-365-days">
+    <!-- Use database-agnostic function to add 365 days -->
+    <sql>
+      UPDATE user_initial_credits_expiration
+      SET expiration_time = credit_start_time + INTERVAL '365' DAY;
```

Comment:
This will take all expiration times and set them to be 365 days after the corresponding credit start times.

I considered a couple alternatives here:

1. I could add a where clause that only does the above if the difference between the two fields is 120 (the current starting credit validity period). I do not think that this is necessary.
2. I could get rid of the expiration_time column all together. This would mean that expiration time could strictly be controlled by a config value. This would mean that all places where expiration date is mentioned currently would have to be calculated. I could be convinced to go with this approach, but if this is the route that we want to go, I do not think that it is necessary to do immediately.